src/plone/api/content.py

```diff
@@ -569,6 +569,36 @@ def get_uuid(obj=None):
     return IUUID(obj)
 
 
+@required_parameters("obj")
+def get_path(obj=None, relative=False):
+    """Get the path of an object.
+
+    :param obj: [required] Object for which to get its path
+    :type obj: Content object
+    :param relative: Return a relative path from the portal root
+    :type relative: boolean
+    :returns: Path to the object
+    :rtype: string
+    :raises:
+        InvalidParameterError
+    :Example: :ref:`content-get-path-example`
+    """
+    if not hasattr(obj, "getPhysicalPath"):
+        raise InvalidParameterError(f"Cannot get path of object of type {type(obj)}")
+
+    if relative:
+        site = portal.get()
+        site_path = site.getPhysicalPath()
+        obj_path = obj.getPhysicalPath()
+        if obj_path[: len(site_path)] != site_path:
```

Comment:
Minor suggestion.  I think this would be a bit easier to understand (for me anyway):

```suggestion
        if not obj_path.startswith(site_path):
```

Reply:
I thought about using `startswith`, but it would require converting tuples to strings, so I went with the tuple slice approach
  `(obj_path[:len(site_path)] == site_path)`
 to avoid the extra overhead. Let me know what you think.
 I'm making the change regardless. I do agree we should prefer readability here.

Reply:
Indeed, I thought it was a string, sorry!

I think in 99% of the cases that we use `getPhysicalPath` in Plone, we transform it into a string by joining.  Maybe it would have been better if `getPhysicalPath` already gave a string as answer.  But it has been a tuple for the last quarter of a century, so let's leave it. ;-)